src/app/item-card/item-card.component.css

```diff
@@ -0,0 +1,8 @@
+.item-card {
+  width: 300px;
+  height: 500px;
+}
+.item-image {
```

Comment:
If you add the CSS class, `img-fluid`, to the `<img/>` tag in `item-card.component.html`, you won't need this class.